src/libsync/configfile.h

```diff
@@ -60,6 +60,7 @@ class OWNCLOUDSYNC_EXPORT ConfigFile
     [[nodiscard]] bool showConfigBackupWarning() const;
 
     bool exists();
+    bool isWidgetVisible() const;
```

Comment:
I think this is still very vague, it is impossible to tell from the config file which widget is being hidden (I would also say it is misleading as it is in fact multiple widgets being hidden)

Reply:
Okay,
This changes are not specific to only one file, this will be use in entire application where MagentaCloud don’t need specific Widget, and avoid code conflict from .ui files(remove those Widget in MagentaCloud code then will show multiple conflict in .ui files) during merge process, so we mutually decide to hide those object from .cpp code , that’s why the name is generic for only this purpose.

Reply:
Even for your case this boolean is not clear... Which widget(s) are being hidden? How can this be understood from the ConfigFile?

Reply:
Okay.
Any better suggestion from your side? else we have to add multiple variable & function to specific widget.

Reply:
Something like `ncSettingsWidgetsVisible` would be a lot better